core/src/Kokkos_Core_fwd.hpp

```diff
@@ -196,7 +196,7 @@ using SharedSpace = Experimental::SYCLSharedUSMSpace;
 #define KOKKOS_HAS_SHARED_SPACE 1
```

Comment:
Would you remind me why we gave this a value instead of using a simple
```
#define KOKKOS_HAS_SHARED_SPACE
```
?

Reply:
my reasoning was to allow using it inside an `if` ... even though we have the `constexp` var

https://godbolt.org/z/TdoWfrMoG

Reply:
I don't like it but this was already there.  Maybe we can discuss at the developer meeting today and possibly update in a follow up PR.

Reply:
Sounds good.